app/deepseg.cc

```diff
@@ -485,22 +496,22 @@ int main(int argc, char* argv[]) try {
 	}
 
 	// Load background if specified
-	auto pbk((s_backg.empty()) ? nullptr : load_background(s_backg.c_str(), debug));
+	auto pbk(s_backg ? load_background(s_backg.value(), debug) : nullptr);
 	if (!pbk) {
-		if (!s_backg.empty()) {
+		if (s_backg) {
 			printf("Warning: could not load background image, defaulting to green\n");
 		}
 	}
 	// default green screen background
 	cv::Mat bg = cv::Mat(height,width,CV_8UC3,cv::Scalar(0,255,0));
 
-	int lbfd = loopback_init(vcam,width,height,debug);
+	int lbfd = loopback_init(s_vcam.c_str(),width,height,debug);
```

Comment:
What about carrying the STL type over to `loopback_init`?

Reply:
Yep, will do.